python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -955,12 +1147,35 @@ def select_edges(self, expr):
 
         Returns
         -------
-        PropertySelection instance to be used for calls to extract_subgraph()
-        in order to construct a Graph containing only specific edges.
+        PropertySelection
+            Can be used for calls to extract_subgraph()
+            in order to construct a Graph containing only specific edges.
 
         Examples
         --------
-        >>>
+        >>> import cudf
+        >>> import cugraph
+        >>> from cugraph.experimental import PropertyGraph
+        >>> from cugraph.experimental import PropertyGraph
+        >>> df = cudf.DataFrame(columns=["src", "dst", "some_property"],
+        ...                              data=[(99, 22, "a"),
+        ...                                    (98, 34, "b"),
+        ...                                    (97, 56, "c"),
+        ...                                    (96, 88, "d"),
+        ...                                  ])
+        >>> pG = PropertyGraph()
+        >>> pG.add_edge_data(df, type_name="etype", vertex_col_names=("src", "dst"))
+        >>> vert_df = cudf.DataFrame({"vert_id": [99, 22, 98, 34, 97, 56, 96, 88],
+        ...                           "v_prop": [1 ,2 ,3, 4, 5, 6, 7, 8]})
+        >>> pG.add_vertex_data(vert_df, type_name="etype", vertex_col_name="vert_id")
```

Comment:
Same issue here: probably not a good idea to name it "etype" for vertex data. Same for all others like this.

Reply:
Fixed